_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,366 @@
+    <th>Lead</th>
+    <th>Contributor(s)</th>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Architecture</b></td>
+    <td>Finalize endorsement of ESDC OSS Management Framework</td>
+    <td>Endorse the implementation of the framework (already in progress)</td>
+    <td><b>EA</b></td>
+    <td>
+    Senior Mgmt<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Update ESDC OSS Management Framework</td>
+    <td>Update ESDC OSS Management Framework to reflect the expected usage and capabilities of each approved VCS</td>
+    <td><b>EA</b></td>
+    <td>
+    DevCoP<br>
+    IT Strategy<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Define best practices and standards</td>
+    <td>Define best practices, standards and procedures for using Git-based version control systems</td>
+    <td><b>DevCoP</b></td>
+    <td>
+      EA<br>
+      Senior Advisors<br>
+      Enterprise Ops<br>
+      BIS<br>
+      APM<br>
+      ITSM<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Operations</b></td>
+    <td>GCcode:<br>
+    Provide official support</td>
+    <td>Determine the most appropriate model for providing official support in GCcode with the current owner (SSC)</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      SSC<br>
+      TBS<br>
+      DevCoP<br>
+      EA<br>
+      IT Strategy<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Organize projects and monitor</td>
+    <td>Organize ESDC projects in GCcode and monitor usage</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Enable connectivity to the Internet</td>
+    <td>Enable projects in GCcode to deploy to public cloud service providers</td>
+    <td><b>DevOps CoE</b></td>
+    <td>
+      DevCoP<br>
+      Cloud CoE<br>
+      SSC<br>
+      Senior Advisors<br>
+      IT Security<br>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2"><b>Development</b></td>
+    <td>GCcode:<br>
+    Enable DevOps capabilities</td>
+    <td>
+    Ensure DevOps capabilities are available in GCcode, such as:
+    <ul>
+      <li>Continuous Integration (CI)</li>
+      <li>Continuous Deployment (CD)</li>
+    </ul>
+    </td>
+    <td><b>DevOps CoE</b></td>
+    <td>
+    SSC<br>
+    DevCoE<br>
+    ?
+    </td>
+  </tr>
+  <tr>
+    <td>Migrate projects to GCcode</td>
+    <td>Migrate software-related projects, scripts and system/database configurations to the chosen VCS, and gradually sunset legacy version control systems
+    </td>
+    <td><b>Development teams</b></td>
+    <td>
+    DevCoP<br>
+    Operations teams<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="1"><b>Communications</b></td>
+    <td>Train staff</td>
+    <td>Offer training resources to IITB staff related to DevOps and VCS</td>
+    <td><b>Learning Branch</b></td>
+    <td>
+    DevCoP<br>
+    ?
+    </td>
+  </tr>  
+</table>
+
+## Measuring the Strategy's success
```

Comment:
Some DORA-related capabilities measurement could be interesting here:
https://www.devops-research.com/research.html#capabilities

Reply:
Good idea! Added DORA metrics :)

Reply:
Good idea! I added DORA VCS metrics for now :)